swe-challenge.md

```diff
@@ -0,0 +1,76 @@
+# WATTx Software Engineer Challenge
+
+
+## Heating Control
+
+
+For this task, you will need to prototype a simple heating control service.
+
+
+Imagine that you have a single room where you can control a radiator valve and a few temperature sensors installed there.
+
+
+The temperature sensors send the readings periodically to the mqtt topic.
+
+
+Your task is to keep the room temperature at 22C (as indicated by sensor readings) by setting the valve from 0 (fully closed) to 100 (fully open).
```

Comment:
> (as indicated by sensor readings)

Part is a bit misleading imo, as it suggest sensors would be detecting 22C all the time? How about changing this sentence to sth like

> Your task is to keep the room temperature at 22C by setting the valve openness from 0 (fully closed) to 100 (fully open). The current room temperature is indicated by sensor readings.